Changelog.md

```diff
@@ -8,6 +8,7 @@ Features:
  * General: Limit the number of errors output in a single run to 256.
  * General: Support accessing dynamic return data in post-byzantium EVMs.
  * Interfaces: Allow overriding external functions in interfaces with public in an implementing contract.
+ * JSON AST: Use short string representation for TypePointers.
```

Comment:
This should be a bugfix, and perhaps also phrased differently for the changelog: Remove storage qualifier for type name strings.